### DISCUSSION TITLE
Circular Reference Error

### DISCUSSION THREAD

**Discussion by ahs53** [gonzalezreal/swift-markdown-ui#326] [General]
Hi, and thanks for this package -- it works great.

I just updated to MacOS Seqoia and Xcode 16, and on compilation, I get a Circular Reference Error on the `InlineNode` class.

Anyone else in the same boat and found a fix?

**Comment by gonzalezreal** [comment]
Hi @ahs53,
This is being addressed in #325. Looks like a Swift compiler bug.
